flocker/provision/test/test_digitalocean.py

```diff
@@ -0,0 +1,67 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Tests for ``flocker.provision._digitalocean``.
+"""
+import copy
+import os
+
+import pyocean
+
+from twisted.trial.unittest import SynchronousTestCase, SkipTest
+
+from flocker.provision._digitalocean import (
+    set_latest_droplet_kernel, retry_if_pending)
+from flocker.testtools import random_name
+
+
+TESTING_DROPLET_ATTRIBUTES = {
+    'region': 'lon1',
+    'size': '8gb',
```

Comment:
Is this just a random choice or have you found, for example, that a 512mb droplet is not suitable?